presto-orc/src/main/java/com/facebook/presto/orc/OrcReaderConfig.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDataSize;
+import io.airlift.units.MinDuration;
+
+import static io.airlift.units.DataSize.Unit.BYTE;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class OrcReaderConfig
```

Comment:
Many ORC reader configs are merged with `HiveClientConfig` (e.g., `orcMaxMergeDistance`). Do we wanna consolidate these two or merge the reader configs to this one. Either should do.

Reply:
I created this new config just so that I could avoid introducing it twice in both `HiveClientConfig` and `StorageManagerConfig` (raptor). Then I thought that we might be able to move the shared config into the new one in orc, but they don't seem to be as compatible.

How about just renaming this into `OrcCacheConfig` :)

Reply:
OrcCacheConfig sounds good to me